models/articleModel.js

```diff
@@ -31,14 +31,34 @@ function fetchArticles() {
       });
   }
 
+  function fetchCommentsById (article_id) {
+    
 
- 
+    return db.query('SELECT * FROM comments WHERE article_id = $1 ORDER BY created_at ASC;', [article_id])
 
+      .then(article => {
+      
+        console.log(article.rows)
 
+        if (article.rows.length === 0) {
```

Comment:
This is great however may result in an error message if the article exists but the comments are empty, this wouldnt be an error case and we'd expect to recieve an empty array back